src/pages/nft/claim/index.vue

```diff
@@ -13,289 +74,418 @@
           "
           target="_blank"
         >
-          <NFTWidgetContentPreview
-            :class="[
-              'transition-shadow',
-              'cursor-pointer',
-              'min-h-[300px]',
-              'w-full',
-            ]"
-            :title="NFTName"
-            :description="NFTDescription"
-            :img-src="NFTImageUrl"
-            @click="handleClickViewDetails"
-          />
+          <div class="flex flex-col gap-[16px] mb-[24px]">
+            <NFTBookCoverWithFrame :src="NFTImageUrl" />
+            <Label class="text-[24px] font-600" :text="NFTName" />
+            <div class="grid grid-cols-2">
+              <div v-if="iscnWorkAuthor" class="flex flex-col">
+                <Label
+                  preset="h6"
+                  :text="$t('nft_claim_NFT_author')"
+                  class=" text-medium-gray font-[500]"
+                />
+                <Label preset="h5" class="font-[500]" :text="iscnWorkAuthor" />
+              </div>
+              <div v-if="creatorDisplayName" class="flex flex-col">
+                <Label
+                  preset="h6"
+                  :text="$t('identity_type_publisher')"
+                  class=" text-medium-gray font-[500]"
+                />
+                <Label
+                  preset="h5"
+                  class="font-[500]"
+                  :text="creatorDisplayName"
+                />
+              </div>
+            </div>
+            <p class="w-full text-[14px] line-clamp-3 font-[400]">
+              {{ NFTDescription }}
+            </p>
+          </div>
         </NuxtLink>
       </NFTWidgetBaseCard>
-    </div>
+      <NFTClaimMainSection
+        v-if="state === NFT_CLAIM_STATE.ERROR"
+        :key="state"
+        :header-text="`ERRORs`"
```

Comment:
Translate?

Reply:
[🌐 Add i18n to stepper & error status](https://github.com/likecoin/liker-land/pull/1588/commits/a2593a3add3ffc223c782c2fcdae8b0cf8a2eb4e)